app/src/androidTest/java/org/mozilla/fenix/ui/robots/SitePermissionsRobot.kt

```diff
@@ -13,19 +13,18 @@ import org.junit.Assert.assertFalse
 import org.junit.Assert.assertTrue
 import org.mozilla.fenix.R
 import org.mozilla.fenix.helpers.TestAssetHelper.waitingTime
+import org.mozilla.fenix.helpers.TestHelper
 import org.mozilla.fenix.helpers.TestHelper.getPermissionAllowID
 import org.mozilla.fenix.helpers.TestHelper.packageName
 import org.mozilla.fenix.helpers.click
 
 class SitePermissionsRobot {
-    fun clickAppPermissionButton(allow: Boolean) {
-        if (allow) {
-            allowSystemPermissionButton.waitForExists(waitingTime)
-            allowSystemPermissionButton.click()
-        } else {
-            denySystemPermissionButton.waitForExists(waitingTime)
-            denySystemPermissionButton.click()
-        }
+    fun clickGrantAppPermissionButton() {
+        TestHelper.grantPermission()
+    }
+
+    fun clickDenyAppPermissionButton() {
+        TestHelper.denyPermission()
```

Comment:
See my comments below about using this method to keep two tests. Otherwise, this is not used anywhere and in that case, could be deleted?